kitsune/settings.py

```diff
@@ -920,6 +920,7 @@ def JINJA_CONFIG():
 CELERY_WORKER_CONCURRENCY = config('CELERY_WORKER_CONCURRENCY', default=4, cast=int)
 CELERY_TASK_EAGER_PROPAGATES = config('CELERY_TASK_EAGER_PROPAGATES', default=True, cast=bool)  # Explode loudly during tests.
 CELERY_WORKER_HIJACK_ROOT_LOGGER = config('CELERY_WORKER_HIJACK_ROOT_LOGGER', default=False, cast=bool)
+CELERY_WORKER_PREFETCH_MULTIPLIER = config('CELERY_WORKER_PREFETCH_MULTIPLIER', default=1, cast=int)
```

Comment:
What's the rationale behind this? Usually we don't have tasks that consume too much time. Have some prefetching should help (eg the default value). Especially for this fxa queue where tasks are small, limiting it to `1` probably won't help.

Reply:
This was to ensure high priority tasks won't be blocked by long-running prefetched low-priority tasks, maybe less necessary now that we have a separate queue for fxa tasks, however we still have those very long-running rebuild_chunk tasks. Tuning is probably necessary here, which is why I added it as an env var.

Reply:
Actually this is just a config. I would suggest to other remove it completely or change the default to the default celery value (that should 4 I believe)